rumqttc/src/websockets.rs

```diff
@@ -8,6 +10,33 @@ pub enum UrlError {
     Parse(#[from] http::uri::InvalidUri),
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum ValidationError {
+    #[error("Websocket response does not contain subprotocol header")]
+    SubprotocolHeaderMissing,
+    #[error("Websocket subprotocol header: {0}")]
+    SubprotocolMqtt(String),
```

Comment:
can we rename this variant to something that will indicate that we can't find "mqtt" in subprotocols list?